src/Command/createArch.php

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use Centreon\Command\CreateCoreArchCommand;
+use Centreon\Command\Service\CreateCoreArchCommandService;
+use Centreon\Command\Service\CreateCoreCommandArchCommandService;
+use Centreon\Command\Service\CreateCoreQueryArchCommandService;
+use Symfony\Component\Console\Application;
+
+$application = new Application();
+$config = yaml_parse_file(__DIR__ . '/config.yaml');
```

Comment:
`yaml_parse_file` needs the yaml extension : I am not sure we all have it.

Reply:
Done